### ISSUE TITLE
Having one vector column for multiple text columns on Qdrant

### ISSUE THREAD

**Issue by vhdmoradi** [qdrant/qdrant#3746]
I have a products table that has a lot of columns, which from these, the following ones are important for our search:

 1. Title 1 to Title 6 (title in 6 different languages)
 2. Brand name
 3. Category name
 4. Product attributes like size, color, etc.

We are planning on using qdrant vector search to implement fast vector queries. But the problem is that all the data important for searching, are in different columns (6 languages + brand + category + attrs = 9 columns at least) and I do not think generating vector embeddings separately for all the columns is the best solution. 

I came up with the idea of mixing the columns together; and I came up with this solution because the titles are essentially the same just in different langs, and the category, brand and attrs columns are all in English language. 

Also I use the "BAAI/bge-m3" model which is a multilingual text embedding model that supports more than 100 langs.

So, in short, I have a vector column containing the vector for the combined text of 6 titles, brand, and category. 

Now, the question is, is this a valid method? What are the pros and cons of this method? 

It is important to note that currently we have more than 300.000 products and they will grow to more than 1.000.000 in the near future.

**Comment by generall** [comment]
Overall mixing different columns in one vector fields should be fine. At least for different titles. But having category name embedded might produce a lot of noise of duplicated vectors, which is not good for vector index.
